paper/paper.md

```diff
@@ -11,36 +11,57 @@ authors:
 
-# Statement of need
+Much of the past work in evaluating equity and resilience have looked at historic data. For instance *<u>(XXX need to add a sentence for each of a 2+ of additional pieces of work, a place to start might be reviewing the citations in [@lin_equity-based_2022])</u>* In another example Brockway, et al. [@brockway_inequitable_2021] show how existing grid infrastructure limits may limit disadvantage community ability to charge electric vehicles and their access to DERs by looking at the geospatial intersection of demographics and grid hosting capacity data. ERAD instead provides a way to look ahead at possible future outages and service access challenges.
+
+Many of the software tools developed for forward-looking understanding of grid resilience have been focused on the bulk power system <u>*(XXX Need to add 2+ citations to support this)*</u>, but this larger scale cannot capture critical service access or outage impacts at the neighborhood, let alone individual customer scale. Existing bulk scale tools, which focus on the integrity of the high-voltage regional, national, and continental scale grid also can not adequately capture the local resilience and equity values of distributed energy resources (DERs) that can provide power to critical service facilities and keep the lights on for customers even when there a challenges getting power from the bulk system. ERAD's enables capturing these local effects at the neighborhood level or even down to the customer level.
+
+There have been some recent efforts that consider resilience at the distribution system. For instance the REPAIR tool [@repair-paper] optimizes distribution system expansion planning considering both routine operations reliability and resilience to extreme events. However, it requires users to supply outage rate information and does not consider any equity metrics or look across critical services. The ReNCAT tool [@wachtel_rencat_2022] does consider multi-service-based equity using the social burden metric introduced in Wachtel, et al. [@wachtel_measuring_2022] (also available as an option in ERAD), however it only considers microgrids as a resilience strategy and it's optimization-based approach may struggle to scale to the very large regions ERAD can evaluate. ReNCAT also currently requires the .Net framework (nominally under the Windows operating system) and though open source, does not currently maintain a publicly accessible code repository to enable outside contributions or issue reporting. Both REPAIR and ReNCAT also require exogenous equipment damage risk data and patterns. In contrast, ERAD endogenously models damage patterns; computes a range of equity metrics, including multi-service access social burden; and enables open engagement in development through github.com.
+
+# Implementation Overview
```

Comment:
I added and expanded this section to provide a bit more depth to the high level description of what is going on. Somewhat similar depth seems to be common in many of the JOSS articles, especially those that are highly cited. To me the idea is for the paper to stand alone with enough description for users to know if they want to look in more detail, such as to start digging into the documentation. If it is too terse, some would be users may never go to the repo--or e.g. a PI may not ask those more comfortable in code to check it out.